diff --git a/node_modules/gatsby-source-medium/.babelrc b/node_modules/gatsby-source-medium/.babelrc
old mode 100644
new mode 100755
diff --git a/node_modules/gatsby-source-medium/CHANGELOG.md b/node_modules/gatsby-source-medium/CHANGELOG.md
old mode 100644
new mode 100755
diff --git a/node_modules/gatsby-source-medium/LICENSE b/node_modules/gatsby-source-medium/LICENSE
old mode 100644
new mode 100755
diff --git a/node_modules/gatsby-source-medium/README.md b/node_modules/gatsby-source-medium/README.md
old mode 100644
new mode 100755
diff --git a/node_modules/gatsby-source-medium/gatsby-node.js b/node_modules/gatsby-source-medium/gatsby-node.js
old mode 100644
new mode 100755
index 3ab21c6..a78374d
--- a/node_modules/gatsby-source-medium/gatsby-node.js
+++ b/node_modules/gatsby-source-medium/gatsby-node.js
@@ -1,10 +1,16 @@
 "use strict";
 
 const axios = require(`axios`);
+const fs = require('fs');
 
 const fetch = (username, limit = 100) => {
-  const url = `https://medium.com/${username}/latest?format=json&limit=${limit}`;
-  return axios.get(url);
+  // const url = `https://medium.com/${username}/latest?format=json&limit=${limit}`;
+  return new Promise(resolve =>
+    fs.readFile('medium-posts.txt', 'utf8', (err, data) => {
+      if (err) throw err;
+      resolve({ data });
+    })
+  );
 };
 
 const prefix = `])}while(1);</x>`;
diff --git a/node_modules/gatsby-source-medium/index.js b/node_modules/gatsby-source-medium/index.js
old mode 100644
new mode 100755
